docs/en/user/upgrade-2.0.x-2.1.0.rst

```diff
@@ -0,0 +1,50 @@
+Upgrading from 2.0.x to 2.1.0
+=============================
+
+.. warning::
+Before this migration, if you configured the Pocket import by adding your consumer key in Internal settings, please do a backup of it: you'll have to add it into the Config page after the upgrade.
+
+Upgrade on a dedicated web server
+---------------------------------
+
+The last release is published on https://www.wallabag.org/pages/download-wallabag.html. In order to upgrade your wallabag installation and get the last version, run the following commands in you wallabag folder (replace ``2.1.0`` by the last release number):
+
+::
+
+    git fetch origin
+    git fetch --tags
+    git checkout 2.1.0
+    ASSETS=build ./install.sh
+    php bin/console doctrine:migrations:migrate --env=prod
+    php bin/console cache:clear --env=prod
+
+Upgrade on a shared hosting
+---------------------------
+
+Backup your ``app/config/parameters.yml`` file.
+
+Download the last release of wallabag:
+
+.. code-block:: bash
+
+    wget http://wllbg.org/latest-v2-package && tar xvf latest-v2-package
+
+(md5 hash of the package: ``4f84c725d1d6e3345eae0a406115e5ff``)
+
+Extract the archive in your wallabag folder and replace ``app/config/parameters.yml`` with yours.
```

Comment:
Be careful, because we added new stuff in this file: https://github.com/wallabag/wallabag/blob/2.1/app/config/parameters.yml.dist#L44-L52